node/src/database/db_migrations/migrator_utils.rs

```diff
@@ -176,9 +179,16 @@ pub struct StatementWithRusqliteParams {
 }
 
 impl StatementObject for StatementWithRusqliteParams {
-    fn execute(&self, transaction: &Transaction) -> rusqlite::Result<()> {
+    fn execute(&self, transaction: &dyn TransactionWrapper) -> rusqlite::Result<()> {
```

Comment:
The argument name is `transaction`, I believe we should rename it to `transaction_wrapper`.

Reply:
You know what is a wrapper, right? It should do just what the wrapped thing inside already does. 

I'm used to seeing: conn or connection wherever we use the ConnectionWrapper. It has never been weird to me to interchange the two things completely, because it actually is almost an identical thing. Plus, we can avoid unnecessarily long names.